ReactiveTask.xcodeproj/project.pbxproj

```diff
@@ -389,7 +389,7 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				GCC_NO_COMMON_BLOCKS = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Instead of changing the deployment target for the project/framework, can you change it just for the test target?